schemas/file.yml

```diff
@@ -184,3 +184,21 @@
         https://www.iana.org/assignments/media-types/media-types.xhtml[IANA official types],
         where possible. When more than one type is applicable, the most specific
         type should be used.
+
+    - name: fork_name
+      level: extended
+      type: keyword
+      description: >
+        A fork is additional data associated with a filesystem object.
+
+        On Linux, a resource fork is used to store additional data with a filesystem object. A file always has at
+        least one fork, for the data portion and additional forks may exist.
+
+        On Windows, this is analogous to an Alternate Data Stream (ADS), and the default data stream for a file is just called $DATA.
+        Zone.Identifier is commonly used by Windows to track contents downloaded from the Internet. An ADS is typically of the form:
+        `C:\path\to\filename.extension:some_fork_name`, and `some_fork_name` is the value that should populate `fork_name`.
+        `filename.extension` should populate `file.name`, and `extension` should populate `file.extension`.
+        The full path, `file.path` will include the fork name.
```

Comment:
```suggestion
        The full path, `file.path`, will include the fork name.
```

Reply:
@rw-access are you happy with the suggested changes from the last review?

If you run `make` locally and `commit` the changed artifact files this can often resolve the failed checks